docs/blog/2020-02-17-changing-the-oss-status-quo/index.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
You could add OSS to `typo-ci.yml` file for the filename, since that will likely come up again in the future.